llvm/tools/llc/llc.cpp

```diff
@@ -537,9 +537,7 @@ static int compileModule(char **argv, LLVMContext &Context) {
       // If we are supposed to override the target triple, do so now.
       std::string IRTargetTriple = DataLayoutTargetTriple.str();
       if (!TargetTriple.empty())
-        IRTargetTriple = TargetTriple;
-      if (!IRTargetTriple.empty())
-        IRTargetTriple = Triple::normalize(IRTargetTriple);
+        IRTargetTriple = Triple::normalize(TargetTriple);
```

Comment:
Is this change still needed? Is it possible to avoid making changes to this file?

Reply:
No. There's no change in llc.cpp now.